### ISSUE TITLE
Block checksum mismatch

### ISSUE THREAD

**Issue by Cryptec** [iotaledger/iri#1344]
Since the Update to IRI 1.6.0 i get almost daily an issue and my Fullnode stop running. 

### Bug description
The first time the error has appeared during a spam, since then, every time i check my Node stats, the milestones stuck and i have to restart IRI to sync the Node Again. I have 6 reliable Neighbours.

### IRI version
IRI 1.6.0 - RELEASE

### Hardware Spec
VPS with:
6 CPU vCores
12 GB RAM
600 GB SSD
500 MBit/s
Traffic Unlimited

### Steps To Reproduce
i don´t know how to reproduce this issue, it happens every view hours without do anything. 

### Errors
 01/20 11:25:32.087 [pool-12-thread-4] ERROR com.iota.iri.network.Node - Error accessing persistence store.
 01/20 11:25:32.182 [Tip Solidity Rescan] ERROR com.iota.iri.service.TipsSolidifier - Error during solidity scan : {}
Jan 20 11:25:32 h2791702.stratoserver.net iri[19535]: org.rocksdb.RocksDBException: block checksum mismatch: expected 1944684774, got 1313198687  in mainnetdb/214752.sst offset 45617113 size 2351

**Comment by GalRogozinski** [comment]
> org.rocksdb.RocksDBException: block checksum mismatch:
Sounds like your DB is corrupted

**Comment by Cryptec** [comment]
ok, i reload the DB. lets see if it solves the Problem. thank you

**Comment by GalRogozinski** [comment]
I am leaving this issue so that others that experienced this issue can comment. Currently I don't know whether it is caused by a bug in IRI or because of a faltering I/O on users' machines.

**Comment by Cryptec** [comment]
short Update, it happens again with the Fresh DB. I try to find an issue now. At the Moment, everything looks fine

**Comment by damageco** [comment]
I've got the same problem. I'm was using 1.8.4, I've upgraded to 1.8.5 yesterday, but the error always appears after of few hours.
My Hardware Spec : 
Bare metal machine. 
8 CPU cores i7
8 GB RAM
1To + 2To disk
1 GBit/s
Traffic Unlimited

I've tried to change the Linux distribution, RedHat, Ubuntu, same problem.

Got this stacktrace : ERROR ReceivedStage:57 - error persisting newly received tx
org.rocksdb.RocksDBException: block checksum mismatch: expected 1900677990, got 1492066657  in /iri/data//002100.sst offset 26773563 size 1276
